src/change/sdp.rs

```diff
@@ -143,7 +151,18 @@ impl<'a> SdpApi<'a> {
             return Err(RtcError::ChangesOutOfOrder);
         }
 
-        add_ice_details(self.rtc, &answer)?;
+        let new_ice_creds = pending.changes.iter().find_map(|c| match c {
+            Change::IceRestart(c) => Some(c),
+            _ => None,
+        });
+
+        // The remote side indicates acceptance by changing its ICE credentials
+        let ice_restart = self
+            .rtc
+            .ice
+            .remote_credentials_changed(answer.ice_creds().as_ref());
+
+        add_ice_details(self.rtc, &answer, ice_restart, new_ice_creds)?;
```

Comment:
Applying new ice creds should move to `apply_offer()` and `apply_answer()`, which is where we effectuate changes of offers/answers.

Reply:
Did you decide if we should still move this?

Reply:
Happy with this for now.